### ISSUE TITLE
Quick questions

### ISSUE THREAD

**Issue by DEV-idiotXD** [ShadowMario/FNF-PsychEngine#1932]
### What is your question?

Im currently working on a mod and i have a few question

1. (Forget make strum the same i found a discussion)

2. How do i make a option and make it affect the game
3. How do i make entering story mode goto a week (the one shadow sent didnt work)

**Comment by ShadowMario** [comment]
Need to know if it's Lua or not, bruh.

**Comment by DEV-idiotXD** [comment]
Source ofc u cant do any of this with lua

**Comment by DEV-idiotXD** [comment]
U cant run lua scripts in the main menu and im using 0.4.2

**Comment by NotEpik** [comment]
I just did the option affect the game a few days ago. First, you additional options in the options menu and then get to save in ClientPrefs something idk and then make a name or idk for it and use the `if (ClientPrefs.OptionsAffactGamesName)` <- this is if the options are true or false but idk how to do other options like scroll speed thing, so if it a true it affects a thing if it a false it won't affect thing in-game. I did an options to choose if you want to use mods's custom scoreTxt thing or the original scoreTxt, and it did work but if you want to use like scroll speed thing i don't know then.
